File: app/src/main/java/com/app/rupiksha/models/ModelUserInfo.java
package com.app.rupiksha.models;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ModelUserInfo {

    @SerializedName("username")
    @Expose
    private String username;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("mobile")
    @Expose
    private String mobile;
    @SerializedName("email")
    @Expose
    private String email;
    @SerializedName("profile")
    @Expose
    private String profile;

    @SerializedName("itr_register")
    @Expose
    public String itrRegister;
    @SerializedName("itr_login")
    @Expose
    public String itrLogin;

    @SerializedName("outlet")
    @Expose
    public String outlet;

    public String getOutlet() {
        return outlet;
    }

    public String getItrRegister() {
        return itrRegister;
    }

    public void setItrRegister(String itrRegister) {
        this.itrRegister = itrRegister;
    }

    public String getItrLogin() {
        return itrLogin;
    }

    public void setItrLogin(String itrLogin) {
        this.itrLogin = itrLogin;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getMobile() {
        return mobile;
    }

    public void setMobile(String mobile) {
        this.mobile = mobile;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getProfile() {
        return profile;
    }

    public void setProfile(String profile) {
        this.profile = profile;
    }
}

File: app/src/main/java/com/app/rupiksha/models/ModelUserInfo.kt
package com.app.rupiksha.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ModelUserInfo(
    @SerializedName("username")
    @Expose
    val username: String? = null,
    
    @SerializedName("name")
    @Expose
    val name: String? = null,
    
    @SerializedName("mobile")
    @Expose
    val mobile: String? = null,
    
    @SerializedName("email")
    @Expose
    val email: String? = null,
    
    @SerializedName("profile")
    @Expose
    val profile: String? = null,

    @SerializedName("itr_register")
    @Expose
    val itrRegister: String? = null,
    
    @SerializedName("itr_login")
    @Expose
    val itrLogin: String? = null,

    @SerializedName("outlet")
    @Expose
    val outlet: String? = null
)
